sound/soc/sof/loader.c

```diff
@@ -87,6 +87,175 @@ int snd_sof_fw_parse_ext_data(struct snd_sof_dev *sdev, u32 bar, u32 offset)
 }
 EXPORT_SYMBOL(snd_sof_fw_parse_ext_data);
 
+/*
+ * IPC Firmware ready.
+ */
+static void sof_get_windows(struct snd_sof_dev *sdev)
+{
+	struct sof_ipc_window_elem *elem;
+	u32 outbox_offset = 0;
+	u32 stream_offset = 0;
+	u32 inbox_offset = 0;
+	u32 outbox_size = 0;
+	u32 stream_size = 0;
+	u32 inbox_size = 0;
+	int mbox_offset;
```

Comment:
@dbaluta looks good to me now but for this minor change. Better to rename this to elem_offset instead of mbox_offset.

Reply:
What about maybe window_offset instead of elem_offset. Also, maybe rename snd_sof_dsp_get_window_elem_offset to snd_sof_dsp_get_window_offset.

As far as I understand, each of our IPC regions is resides in a window. In order to get the address of the region we need:

* base address + offset of window + offset of elem inside window.

Reply:
@dbaluta yes makes sense